front/lib/api/assistant/conversation.ts

```diff
@@ -677,16 +701,29 @@ export async function* postUserMessage(
   }
 
   const results = await Promise.all([
-    removeNulls(
-      await Promise.all(
-        mentions.filter(isAgentMention).map((mention) => {
-          return getLightAgentConfiguration(auth, mention.configurationId);
-        })
-      )
+    await Promise.all(
```

Comment:
The `await` here breaks the || of the `Promise.all()`, right? IIRC `Promise.all()` should take promises, here we await them.

Reply:
:+1: well spotted, we did had the issue before too I guess ; I refactored without noticing